integration/step-definitions/evaluation.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { defineFeature, loadFeature } from 'jest-cucumber';
+import { OpenFeature } from '../../src/open-feature.js';
+
+// load the feature file.
+const feature = loadFeature('integration/features/evaluation.feature');
+
+// get a client (flagd provider registered in setup)
+const client = OpenFeature.getClient();
+
+defineFeature(feature, (test) => {
```

Comment:
These tests are associated with the gerkin. If any test descriptions don't line up with the gerkin clauses, the test fails.